dotnetv3/S3/README.md

```diff
@@ -0,0 +1,105 @@
+- [GenPresignedUrlExample](GenPresignedUrlExample/) (`GetPreSignedURLAsync`)
+- [GetObjectExample](GetObjectExample/) (`GetObjectAsync`)
+- [LifecycleExample](LifecycleExample/)
+(`GetBucketLifecycleConfigurationAsync`, `PutBucketLifecycleConfigurationAsync`, `DeleteLifecycleConfigurationAsync`)
+- [ListBucketsExample](ListBucketsExample/) (`ListBucketsAsync`)
+- [ListObjectsExample](ListObjectsExample/) (`ListObjectsV2Async`)
+- [ListObjectsPaginatorExample](ListObjectsPaginatorExample/) (`ListObjectsV2Async`)
+- [ManageACLsExample](ManageACLsExample/) (`PutBucketAsync`, `GetACLAsync`, `PutACLAsync`)
+- [ManageObjectACLExample](ManageObjectACLExample/) (`PubtObjectACLAsync`, `GetObjectAclAsync`)
+- [MPUapiCopyObjExample](MPUapiCopyObjExample/)
+(`InitiateMultipartUploadAsync`, `CopyPartAsync`, `CompleteMultipartUploadAsync`)
+- [DeleteMultipleObjectsExample](non-versioned-examples/DeleteMultipleObjectsExample/) (`DeleteObjectsAsync`)
+- [DeleteObjectExample](non-versioned-examples/DeleteObjectExample/) (`DeleteObjectAsync`)
+- [ObjectTagExample](ObjectTagExample/) (`PutOjectAsync`, `PutObjectTaggingAsync`)
+- [RestoreArchivedObjectExample](RestoreArchivedObjectExample/) (`RestoreObjectAsync`)
+- [S3CORSExample](S3CORSExample/) (`GetBucketCorsAsync`, `PutBucketCorsAsync`, `DeleteBucketCorsAsync`)
+- [ServerAccessLoggingExample](ServerAccessLoggingExample/) (`PutACLAsync`, `PutBucketLoggingAsync`)
+- [ServerSideEncryptionExample](ServerSideEncryptionExample/) (`PutObjectAsync`, `GetObjectMetadataAsync`)
+- [SSEClientEncryptionExample](SSEClientEncryptionExample/) (`PutObjectAsync`, `GetObjectAsync`, `GetObjectMetadataAsync`)
+- [SSECLowLevelMPUcopyObjectExample](SSECLowLevelMPUcopyObjectExample/)
+(`InitiateMultipartUploadAsync`, `UploadPartAsync`, `CompleteMultipartUploadAsync`)
+- [TempCredExplicitSessionStartExample](TempCredExplicitSessionStartExample/) (`GetSessionTokenAsync`)
+- [TempFederatedCredentialsExample](TempFederatedCredentialsExample/)
+(`GetTemporaryFederatedCredentialsAsync`, `GetFederationTokenAsync`)
+- [TrackMPUUsingHighLevelAPIExample](TrackMPUUsingHighLevelAPIExample/) (`FileTransferUtility.UploadAsync`)
+- [TransferAccelerationExample](TransferAccelerationExample/) (``)
+- [UploadDirMPUHighLevelAPIExample](UploadDirMPUHighLevelAPIExample/) (``)
+- [UploadFileMPUHighLevelAPIExample](UploadFileMPUHighLevelAPIExample/) (``)
+- [UploadFileMPULowLevelAPIExample](UploadFileMPULowLevelAPIExample/) (``)
+- [UploadObjectExample](UploadObjectExample/) (`PutObjectAsync`) (``)
+- [UploadUsingPresignedURLExample](UploadUsingPresignedURLExample/) (`GetPreSignedURLAsync`)
+- [versioned-examples/DeleteMultipleObjectsExample](versioned-examples/DeleteMultipleObjectsExample/) (``)
+- [versioned-examples/DeleteObjectVersionExample](versioned-examples/DeleteObjectVersionExample/)
+(`DeleteObjectVersionAsync`)
+- [versioned-examples/ListObjectVersionsExample](versioned-examples/ListObjectVersionsExample/) (`ListObjectVersionsAsync`)
```

Comment:
Folder prefix looks odd here.

Reply:
Removed the folder from the display but let it in the URL.